static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -111,10 +111,9 @@ const setPublisherProvidedId = (): void => {
     }
 };
 
-export const init = (start: () => void, stop: () => void): Promise<void> => {
+// onFillAdvertsComplete is used by unit tests to wait for fillAdvertSlots
```

Comment:
Was this comment left from something you were trying before?

Reply:
yep, should've cleaned that up!